src/odemis/driver/simstreakcam.py

```diff
@@ -274,7 +274,7 @@ def _getNewImage(self):
         """
         self._img_counter = (self._img_counter + 1) % len(self._img_list)
         image = self._img_list[self._img_counter]
-
+        image += (numpy.random.rand(*image.shape) * 1000 - 500).astype(numpy.uint16)
```

Comment:
Let's not add this extra debug code here. This driver already supports sending multiple images.